src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -2401,6 +2407,29 @@ private bool StartSourceChecksumCalculation(PEModuleBuilder moduleBeingBuilt, Di
                 return false;
             }
 
+            // Add debug documents for all embedded text first. This ensures that embedding takes priority over the 
```

Comment:
Yes. shelved VB and tests during major refactoring. Will port now as things settle.